### PR TITLE
i343_Allow a single number to be entered for the number of additional flow nodes

### LINKED ISSUE #343: Allow a single number to be entered for the number of additional flow nodes
After #342 it is no longer possible to only a dd a single number for additional flow nodes. This forces users to use brackets (a list) even if they don't use multiple layers in flownet.

The part of the code which splits the additional flow nodes based on volume is no longer reachable as the additional flow nodes can never be a number.

### PR DESCRIPTION
**Pull request by LonnekevB**
…ple layers

*Insert a description of your pull request (PR) here, and check off the boxes below when they are done.*

---

### Contributor checklist

- [ ] :tada: This PR closes #343.
- [ ] :scroll: I have broken down my PR into the following tasks:
   - [ ] Task 1
   - [ ] Task 2
- [ ] :robot: I have added tests, or extended existing tests, to cover any new features or bugs fixed in this PR.
- [ ] :book: I have considered adding a new entry in `CHANGELOG.md`.
- [ ] :books: I have considered updating the documentation.

### CODE REVIEW

File: src/flownet/config_parser/_config_parser.py [hunk 1]
```diff
@@ -385,7 +399,10 @@ def _to_abs_path(path: Optional[str]) -> str:
                     },
                     "additional_flow_nodes": {
                         MK.Type: types.List,
-                        MK.Description: "List of additional flow nodes to add for each layer.",
+                        MK.Description: "List of additional flow nodes to add "
+                        "for each layer or single integer which the code will "
```
Comment (wouterjdb) [inline src/flownet/config_parser/_config_parser.py:403]:
```suggestion
                        "for each layer or single integer which will be "
```

File: src/flownet/config_parser/_config_parser.py [hunk 2]
```diff
@@ -31,6 +31,20 @@ def create_schema(config_folder: Optional[pathlib.Path] = None) -> Dict:
 
     """
 
+    @configsuite.transformation_msg("Convert integer to list")
+    def _integer_to_list(input_data: Union[List, int]) -> List:
+        """
+        Converts integer to list with single item.
+        Args:
```
Comment (wouterjdb) [inline src/flownet/config_parser/_config_parser.py:38]:
```suggestion

        Args:
```

**Review by wouterjdb — COMMENTED** [review]
Have you tested the code with a single integer in the config?

**Comment by LonnekevB** [comment]
> Have you tested the code with a single integer in the config?

Yes I did

**Comment by wouterjdb** [comment]
> > Have you tested the code with a single integer in the config?
> 
> @wouterjdb: Yes I did, it works. I guess we can change the ci configs back to single integers. Shall I make a pull request in test-data to do that?

Yes!

**Review by wouterjdb — APPROVED** [review]

**Pull request merged**